src/go/rpk/go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.55.0
-	github.com/redpanda-data/common-go/rpadmin v0.1.2
+	github.com/redpanda-data/common-go/rpadmin v0.1.4
```

Comment:
We had an issue in `dev` with this tag and I had to bump it in https://github.com/redpanda-data/redpanda/pull/23244/files

Can you bump it to v0.1.6 here as well? thanks!

Reply:
Fixed in https://github.com/redpanda-data/redpanda/commit/88b7f7e6c59d73c316c71cfb79b7f5ab876c4a4b. Thank you.